Libplanet/Net/InvalidTimestampException.cs

```diff
@@ -10,7 +11,7 @@ public class InvalidTimestampException : Exception
         internal InvalidTimestampException(
             string message,
             DateTimeOffset createdOffset,
-            TimeSpan lifespan,
+            TimeSpan? lifespan,
```

Comment:
Is it necessary changes? I'm not sure in which case `lifespan` gets the `TimeSpan` as nullable...

Reply:
There is some logic issue in implementations of `IMessageCodec`, which I haven't got my hands on yet. In any case, `lifespan` should be nullable since `InvalidTimestampException` should be throwable even when `lifespan` is `null`.